frontend/src/components/document/DocumentView/StyledText.jsx

```diff
@@ -1,28 +1,27 @@
 import { getCommentThreadsOnTextNode } from "@/util/editorCommentUtils";
+import { getRedactionsOnTextNode, getMarkFromLeaf, replaceRedactionWithX } from "@/util/editorRedactionUtils";
 import CommentedText from "./CommentedText";
+import RedactedText from "../Redactions/RedactedText";
+import RedactionPopover from "../Redactions/RedactionPopover";
+import { accepted, rejected } from "@/assets/redacted_lists";
 
-export default function StyledText({ attributes, children, leaf }) {
+//for table view, pass in false for  isPopoverDisabled to disable popovers
+export default function StyledText({ attributes, children, leaf, isPopoverDisabled }) {
 
-  //TO DO: trim space off of highlight
+  const mark = getMarkFromLeaf(leaf)
 
-  if (leaf.current) {
-    children = <span className="bg-curr-redaction">{children}</span>
+  function removeMark () {  
   }
-
-  if (leaf.suggested) {
-    children = <span className="bg-suggested-redaction">{children}</span>
-  }
-
-  if (leaf.accepted) {
-    children = <span className="bg-accepted-redaction">{children}</span>
-  }
-
-  if (leaf.rejected) {
-    children = <u>{children}</u>
+  //store and send to ML model
+  function onRejectRedaction () {
+    removeMark()
+    rejected.push(leaf)
   }
 
-  if (leaf.bold) {
-    children = <strong>{children}</strong>;
+  //store and send to ML model
+  function onAcceptRedaction () {
+    replaceRedactionWithX(leaf)
```

Comment:
same with this function, since we're changing the "replace with X" behaviour to highlighting in this function anyways (though we'll still need the replacement function for exporting)

Reply:
If we're changing to highlighing then maybe you want to changed the state in slate to be "accepted" rather than removing the mark and just have "accepted" render with the grey highlight? but don't need that for this PR